package.json

```diff
@@ -43,7 +43,8 @@
     "testEnvironment": "node",
     "testPathIgnorePatterns": [
       "/node_modules/",
-      "/fixtures/"
+      "/fixtures/",
+      "/external/"
```

Comment:
Was this actually needed? I think we had to add the `fixtures` folder as it has a `js` file... but don't remember.

Reply:
done!